app/Services/ObjectManager/EloquentObjectManager.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services\ObjectManager;
+
+use Illuminate\Database\Eloquent\Model;
+
+class EloquentObjectManager implements ObjectManagerInterface {
+    public function save($object) {
+        if (!$object instanceof  Model) {
+            throw new \UnexpectedValueException('Expected $object to be of type Model. Got ' . get_class($object));
+        }
```

Comment:
Zou dit in een aparte private method zetten, want DRY

Reply:
Ik zal een apparte exception van maken, maar ik wil liever geen private methode maken die conditioneel throwed omdat dan de control flow minder duidelijk wordt.
Kan je daarmee leven?